apiserver/params/params.go

```diff
@@ -256,6 +256,20 @@ type ApplicationUpdateSeriesArgs struct {
 	Args []ApplicationUpdateSeriesArg `json:"args"`
 }
 
+// MachineUpdateSeriesArg holds the parameters for updating the series for the
+// machine.  Only known by facade version 2 and greater.
+type MachineUpdateSeriesArg struct {
+	MachineName string `json:"name"`
```

Comment:
See earlier comment - we should consider consolidating MachineUpdateSeries and ApplicationUpdateSeries structs. Certainly tags should be used. It's unfortunate that tags are not used consistently.